pkg/agent/openflow/pipeline.go

```diff
@@ -284,6 +284,10 @@ const (
 	// controller send reject packet as packet-out, we want reject response to bypass
 	// the connTrack to avoid unexpected drop.
 	CustomReasonReject = 0b10
+	// CustomReasonDeny is used when sending packet-in message to controller indicating
+	// corresponding connection action is drop or reject, which can be used by Flow
```

Comment:
... indicating that the corresponding connection has been dropped or rejected. It can be consumed by the Flow Exporter to export flow records for connections denied by network policy rules.